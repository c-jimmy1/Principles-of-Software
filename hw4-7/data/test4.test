m
a

s
m
q
