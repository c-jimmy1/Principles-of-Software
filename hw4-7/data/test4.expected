b lists all buildings
r prints directions for the shortest route between any two buildings
m prints a menu of all commandsm
q to quit
Unknown option
Unknown option
Unknown option
b lists all buildings
r prints directions for the shortest route between any two buildings
m prints a menu of all commandsm
q to quit
